app/page.tsx

```
@@ -129,206 +135,293 @@
 
-          <div className="print:hidden text-gray-700 mb-10 flex flex-wrap justify-center gap-2">
-            {topSkills.map((tag, index) => <SkillTag key={index} name={tag} />)}
-          </div>
-        </section>
+        <div className="container mx-auto px-4 max-w-4xl">
 
-        <section id="experience" className="mb-10 print:mb-6">
-          <SectionHeading title="Experience" />
+        <section id="summary" className="cv-section">
+          <SectionHeading title="Summary" subtitle="Professional overview and core competencies" />
+          <p className="text-slate-700 leading-relaxed" dangerouslySetInnerHTML={{ __html: sanitize(userProfile.summary) }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)